### ISSUE TITLE
 hideDefaultLocaleInURL=true then sessionRedirect Problem

### ISSUE THREAD

**Issue by tariksisbacak** [mcamara/laravel-localization#661]
Hi, I have the same problem. read a lot of issues but nothing changed

#650 
#653

```
Route::group(['namespace' => 'Frontend', 'prefix' => LaravelLocalization::setLocale(), 'middleware' => ['web', 'localize', 'localeSessionRedirect', 'localizationRedirect', 'localeViewPath']], function () {

    Route::get('/', [
        'as' => 'frontend.home',
        'uses' => 'Home\HomeController@index'
    ]);

});
```

http://site.test/            (English default)
http://site.test/de

Once the language is switched, the default language is not working.
It redirects to http: //site.test/de instead of http: //site.test/.

"localeSessionRedirect" does not do this if it exists within the route. But when we delete it, access to the default language address becomes available

**Comment by iwasherefirst2** [comment]
Your using `localeSessionRedirect`. This means once your language is set to `de`, every domain without an locale (for example `example.com/about` will be redirected to `example.com/de/about`.
To switch back to English, you should access `example.com/en/about` which should redirect to `example.com/about`. However, this is currently not working in 1.4.2 (see #656). I will submit a fix for that this evening.

**Comment by tariksisbacak** [comment]
I tought the problem was caI thought the problem was caused by the latest release. Because I didn't have this problem in older versions. 1.3. *.

**Comment by iwasherefirst2** [comment]
@tariksisbacak yes exactly. The problem in #656 only occurred since 1.4.2.

**Comment by tariksisbacak** [comment]
The problem still continues. locale does not play default language when I add 'localeSessionRedirect' @iwasherefirst2

**Comment by iwasherefirst2** [comment]
closed again as it was reposted #672
